flight/targets/coptercontrol/board-info/cmsis_system.c

```diff
@@ -201,6 +201,10 @@ static void SetSysClock(void);
   * @{
   */
 
+// this is required for the system_rcc code because
+// of the clock rate switching in naze32

```

Comment:
It's actually needed so that the hse_value can be present on all smallf1 targets, since naze32 needs it.